cmake/tribits/examples/TribitsExampleApp/README.md

```diff
@@ -0,0 +1,110 @@
+# TribitsExampleApp
+
+The example project `TribitsExampleApp` is a raw CMake project that pulls in
+libraries from packages from `TribitsExampleProject`.  To build against all of
+the installed packages from an upstream `TribitsExampleProject`, configure,
+build, and run the tests with:
+
+```
+  cmake \
+    -DCMAKE_PREFIX_PATH=<upstreamInstallDir> \
+    <base-dir>/TribitsExampleApp
+
+  make
+
+  ctest
+```
+
+That will call `find_package(TribitsExaProj)` and will set the compilers
+pulled in from the found `TribitsExaProjConfig.cmake` file.
+
+
+## Pulling in only a subset of packages
+
+To configure and build `TribitsExampleApp` against only a subset of the
+installed `TribitsExampleProject` packages, configure `TribitsExampleApp`
+using, for example:
+
+```
+  cmake \
+    -DCMAKE_PREFIX_PATH=<upstreamInstallDir> \
+    -DTribitsExApp_USE_COMPONENTS=SimpleCxx,WithSubpackages \
+    <base-dir>/TribitsExampleApp
+```
+
+Internally, that that will call:
+
+```
+  find_package(TribitsExApp REQUIRED COMPONENTS ${TribitsExApp_USE_COMPONENTS})
+```
+
+(where `,` in `TribitsExApp_USE_COMPONENTS` is replaced with `;` internally to
+create a proper CMake list internally before calling `find_package()`).
+
+
+## Pulling in only a subset of packages by finding each package individually
+
+`TribitsExampleApp` is also set up to demonstrate finding the individual
+packages using separate calls to `find_package(<Package>)` by configuring
+with, for example:
+
+```
+  cmake \
+    -DCMAKE_PREFIX_PATH=<upstreamInstallDir> \
+    -DTribitsExApp_USE_COMPONENTS=SimpleCxx,WithSubpackages \
+    -DTribitsExApp_FIND_INDIVIDUAL_PACKAGES=ON \
+    <base-dir>/TribitsExampleApp
+```  
+
+That essentially results in calling:
+
+```
+  find_package(SimpleCxx REQUIRED)
+  find_package(WithSubpackages REQUIRED)
+  ...
+  target_link_libraries(app
+    PRIVATE SimpleCxx::all_libs
+    PRIVATE WithSubpackages::all_libs
+    )
+```
+
+(but does so with a loop and a list internally obviously).
```

Comment:
```suggestion
(but does so with a loop and a list internally).
```

Reply:
Thanks! Addressed in TriBITSPub/TriBITS#430